.gitlab-ci.yml

```diff
@@ -21,21 +21,6 @@ init:
   script:
     - schutzbot/update_github_status.sh start
 
-check-build-coverage:
-  stage: verify
-  extends: .terraform
-  variables:
-    RUNNER: aws/fedora-41-x86_64
-    INTERNAL_NETWORK: "true"
-    PYTHONUNBUFFERED: 1
-  script:
-    - sudo ./test/scripts/install-dependencies
-    - ./test/scripts/check-build-coverage ./s3configs/
-  cache:
-    key: testcache
-    paths:
-      - .cache/osbuild-images
-
```

Comment:
You also need to modify https://github.com/osbuild/images/blob/483b2f5226833a0c40f9e176f6a6f63e1296f6db/test/scripts/generate-gitlab-ci#L36-L49